scholarly/_scholarly.py

```diff
@@ -255,13 +258,29 @@ def citedby(self, object: Publication)->_SearchScholarIterator:
         :param object: The Publication object for the bibtex exportation
         :type object: Publication
         """
-        if object['container_type'] == "Publication":
-            publication_parser = PublicationParser(self.__nav)
-            return publication_parser.citedby(object)
-        else:
+
+        if object['container_type'] != "Publication":
             self.logger.warning("Object not supported for bibtex exportation")
             return
 
+        if object["bib"]["citedby"] < 999:
+            return PublicationParser(self.__nav).citedby(object)
+        else:
+            try:
+                year_low = int(object["bib"]["pub_year"])
+                year_end = int(datetime.date.today().year)
+            except:
+                warnings.warn(f"Unknown publication year for paper {object}, may result in incorrect number of citedby papers.")
```

Comment:
You don't want to print the entire `object` here. Just the title extracted from `object` would do.

Reply:
Also, log the warning here.
```suggestion
                self.logger.warning("Unknown publication year for paper %s, may result in incorrect number of citedby papers.", object["bib"]["title"])
```